### ISSUE TITLE
Glass for Modern/Universal apps and other apps

### ISSUE THREAD

**Issue by TrantaLocked** [Maplespe/DWMBlurGlass#311]
**Describe the solution you'd like**

Option to enable glass title bar and Aero button style for Modern/Universal apps like Calculator, Media Player and Settings and selectively per app like for browsers or apps that force their own title bar theme.

**Your reason (tell us why we need it)**
For UI coherence.

**Describe alternatives you've considered**
I have not found solutions elsewhere for this that are up to date.

**Additional context**

![dwmbgpic](https://github.com/Maplespe/DWMBlurGlass/assets/18237916/6ae3efbe-e604-4779-b674-5d978c460c95)

Left shows old apps with glass title bar, right shows modern apps with no glass titlebar but with working glass border.

**Comment by Maplespe** [comment]
A similar issue has been discussed before #194.
But the fact is that these UWP rendering flows are outside the scope of this project, and it is difficult to modify them in DWM.
We may start a new other project to accomplish this, but it will not be implemented in this project.
As for restoring the UWP titlebar, there is a corresponding windhawk mod available.
Forcing other win32 apps to use the background effect MFE can also be done.

**Comment by TheInsane101** [comment]
> We may start a new other project to accomplish this, but it will not be implemented in this project. As for restoring the UWP titlebar, there is a corresponding windhawk mod available.

Yeah, the only issue is that it only gives the close caption button to all the apps and not the window size nor minimise buttons, but I think Dulappy is already working on this as part of his Aero Window Manager project.
